src/elementPlugins/shortText/src/SummaryView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Header, Icon } from 'semantic-ui-react'
+import { SummaryViewProps } from '../../types'
+
+const SummaryView: React.FC<SummaryViewProps> = ({ parameters, value }) => {
+  return (
+    <>
+      <Header as="h3" content={parameters.label} />
+      <p>
+        {!value?.isValid ? <Icon name="exclamation circle" color="red" /> : null}
+        {value?.text}
+      </p>
```

Comment:
```suggestion
<Header as="h3" content={parameters.label} />
      <Input
        disabled
        defaultValue={value?.text}
        icon={!value?.isValid ? <Icon name="exclamation circle" color="red" /> : null}
      />
```

It's better if we always use Semantic-ui element - so it's easier to move on to layout and styling soon.

Reply:
Sure, but this is not an `<Input>` element, and there doesn't seem to be a `<Text>` element like there is in Material-UI. I couldn't really see what the generic "text" element is for Semantic, so assumed there isn't one? Open to suggestions, but don't really think we should be using elements that are semantically incorrect.

Reply:
The benefits of using a existing UI component is that you already get the features of disabling, the icon inside the box, the border. I think the semantic is correct, showing an `Input` disabled for visibility, although I think you are also correct that this is not how it looks in the UI design. I suggest then creating your own component - and using the `Label` or `Header` to display the value filled by the user, and an optional `invalid` icon that shows next to it. So you make your own component, and pass the props to decide how it is presented. That's how I would do it.

Reply:
For the record, here is how the Summary view is currently UI is designed:
<img width="533" alt="Screen Shot 2020-11-09 at 3 28 57 PM" src="https://user-images.githubusercontent.com/16461988/98493833-5ae4fc00-22a0-11eb-80cb-fc02dded0a3a.png">

Reply:
Right, so from that I would interpret it to be that each Section would have a (collapsible) `Container`, each Page a `Segment.Group`,  then each summary item is just a (sub) `<Header>` and a `body` text. Or maybe just bold and standard body text for each item.

FWIW -- all the code examples on the Semantic-UI site just use `<p>` for body text.

But this seems more like a Style discussion, which is maybe a bit premature?

Reply:
No problem to wait for some future only style issue. 

I just thought that making it more similar to existing UI style that we could make (something that doesn't take more than 15 minutes effort) was worth. From my point of view it would also make the code easier to change and read. But don't worry, if you don't feel like doing it for now, it's fine. Was just a suggestion : )

Reply:
I agree in principle, I just think we should probably have a chat and establish some standards before going too far down this road. Thanks